api/src/pcapi/core/offers/commands.py

```diff
@@ -16,7 +16,13 @@
 @blueprint.cli.command("activate_future_offers")
 @log_cron_with_transaction
 def activate_future_offers() -> None:
```

Comment:
```suggestion
@blueprint.cli.command("reindex_recently_published_offers")
@log_cron_with_transaction
def reindex_recently_published_offers() -> None:
```
Je pense que `activate_future_offers` n'a plus trop de sens

Reply:
j'ai un peu hésité, renommer le cron = léger décallage donc 2 commandes cohabitent
Mais ok, c'est pas si lourd